imitation/scripts/data_collect.py

```diff
@@ -16,22 +13,11 @@ def make_PPO2(env_name, num_vec):
   return policy
 
 
-@gin.configurable
+@data_collection_exp.automain
```

Comment:
I ended up opting for just using `@exp.main` in my code and having a `if name == '__main__'` block that calls it, see e.g. https://github.com/HumanCompatibleAI/adversarial-policies/blob/master/src/modelfree/score_agent.py

My reason for this is that you might want to run the experiment from multiple endpoints: as a stand-alone script, or from another script that's running the experiment multiple times (perhaps in parallel) to perform a hyperparameter sweep.

If you use `automain`, then you have to have the same environment (and in particular observers) regardless. If you have a `__main__` block manually defined, you can add the observers and anything else you want in the stand-alone script, leaving the experiment that's exported to other files barebones.

Given that we're not currently using the experiment from multiple files, I think it's fine to keep using `automain`, but just beware you may need to change it later.

Reply:
I used a __main block to easily add an observer